### PR TITLE
Fix various code issues

### PR DESCRIPTION
**Pull request by Jalle19**
Related to #218

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -60,7 +60,6 @@
             "./tests/Functional/Execution/testClasses.php",
             "./tests/Functional/Validation/errors.php",
             "./tests/Functional/Validation/harness.php",
-            "./tests/Functional/QueryTest.php",
```
Comment (crisu83) [inline composer.json]: Good catch.

File: src/Util/ValueHelper.php [hunk 2]
```diff
@@ -50,8 +50,8 @@ function compareArguments(array $argumentsA, array $argumentsB): bool
     }
 
     /**
-     * @param $valueA
-     * @param $valueB
+     * @param ValueNodeInterface $valueA
+     * @param ValueNodeInterface $valueB
      * @return bool
      */
     public function compareValues($valueA, $valueB): bool
```
Comment (crisu83) [inline src/Util/ValueHelper.php]: Could these even be type-hinted? @Jalle19

File: src/Validation/Conflict/Map.php [hunk 3]
```diff
@@ -28,8 +28,8 @@ public function __construct(array $values = [])
 
 
     /**
-     * @param $key
-     * @param $value
+     * @param string $key
+     * @param mixed  $value
      */
     public function set($key, $value): void
```
Comment (crisu83) [inline src/Validation/Conflict/Map.php]: The `$key` could probably be type-hinted. @Jalle19

File: src/Util/ValueHelper.php (hunk 2 above)
Comment (Jalle19) [reply src/Util/ValueHelper.php]: Yes, I had to rebase on master and this change got lost

**Comment by crisu83** [comment]
Rebased on `master`.

**Pull request merged**